### PR TITLE
add: `system76-power-deb`

### PR DESCRIPTION
**Pull request by soumyatheman**

### CODE REVIEW

File: packages/system76-power-deb/system76-power-deb.pacscript [hunk 1]
```diff
@@ -0,0 +1,10 @@
+name="system76-power-deb"
+gives="system76-power"
+breaks="${gives}"
+version="1.1.23"
```
Comment (wizard-28) [inline packages/system76-power-deb/system76-power-deb.pacscript:4]: Outdated version, `1.1.24` is the latest.

**Review by wizard-28 — CHANGES_REQUESTED** [review]

File: packages/system76-power-deb/system76-power-deb.pacscript (hunk 1 above)
Comment (soumyatheman) [reply packages/system76-power-deb/system76-power-deb.pacscript:4]: Not available in the ppa

Comment (wizard-28) [reply packages/system76-power-deb/system76-power-deb.pacscript:4]: Get it from here I guess: https://apt-origin.pop-os.org/release/pool/jammy/system76-power/cee2d561326176396832fa4c1d3edf61556d2dba/

**Comment by soumyatheman** [comment]
It also doesn't works in debian, I guess that may be the reason for not being in the repo.

**Review by wizard-28 — CHANGES_REQUESTED** [review]
Make it `incompatible` with debian bullseye

**Pull request closed without merging**